docs/it/stime.md

```diff
@@ -0,0 +1,74 @@
+
+Per fare un esempio, partendo dal classico to do list, potremmo mettere in risalto autenticazione, le operazioni di base di visione/creazione/modifica/cancellazione di un elemento, a questo aggiungiamo che il nostro committente voglia anche condivisione dell'item e magari memo sul calendario. Ora possiamo individuare l'autenticazione e le operazioni di base come piuttosto ben definite, ma condivisione e integrazione col calendario? Cominciamo a porci delle domande per esempio "condividere con chi" o "quale calendario usare" (Google?). Vediamo quindi emergere dei macro blocchi e degli elementi da chiarire per i quali individuare anche degli esperti sia tecnologici che non che possono aiutarci a farci un'idea di cosa vogliamo ottenere; in tutto questo, quando troviamo uno scoglio nella scomposizione, possiamo in prima battuta impegnare uno slot di tempo per lo studio ben definito il classico "time boxed" oltre il quale non andare e dopo il quale va cercato aiuto.
+
+#### Isolare le criticità
+
+Nella suddivisione si possono scorgere, come si è visto, criticità; sono le parti più complesse da stimare, quelle che ci pongono di fronte a situazioni nuove dove un pattern già conosciuto non è visibile o proprio è assente. Si possono adottare due strategie concatenate: la prima è quella di "assegnare ad una stima del tempo di stima": sembra un gioco di parole ma non lo è. Se un tema è complesso e non scomponibile e ha bisogno di essere studiato è necessario prendersi il tempo per farlo. Assegnare a questo task una stima per consentirci di avere le idee più chiare ci da la maniera di introdurre il concetto di PoC: proof of concept. Di fatto è un micro task orientato alla creazione di uno o più proprietà del progetto finale che sono critiche e che, una volta sbrogliate, rendono tutto il processo di creazione più chiaro. Può anche essere utile mostrarlo, a volte può bastare provare se l'idea funziona a livello tecnico, ma a volte è possibile anche mostrarlo a chi poi prenderà decisioni perché dà immediatamente un'idea di dove si vuole arrivare, che prestazioni o di che interattività si parla.
```

Comment:
suggerisco di fare un elenco puntato con le due strategie in modo da dividere bene il testo (anche visualmente). Inoltre si eviterebbe il ":" dentro un altro ":".